test.py

```diff
@@ -0,0 +1,162 @@
+# pyGene Fusion local testing
+import json
+from pydantic import BaseModel, validator, ValidationError, StrictStr, StrictInt, StrictBool
+from typing import Optional, List, Union
+
+class Gene(BaseModel):
+    type = 'GeneDescriptor'
+    value_id: str
+    label: StrictStr
+
+    @validator('value_id')
+    def is_valid(cls,v):
+        assert v.find(':') != -1 and v.find(' ') == -1, 'id must be a CURIE'
+        return v
+
+class SequenceRegion(BaseModel):
+    type = 'SequenceLocation'
+    seq_id: str
+    start: StrictInt
+    end: StrictInt
+
+    @validator('seq_id')
+    def is_curie(cls, v):
+        assert v.find(':') != -1 and v.find(' ') == -1, 'chr must be a CURIE'
+        return v
+
+class ChromosomeRegion(BaseModel):
+    type = 'ChromosomeLocation'
+    chr: str
+    start: str
+    end: str
+
+    @validator('chr')
+    def valid_chr(cls, v):
+        assert v.isalnum(), 'chr must have only alphanumeric characters'
+        return v
+
+    @validator('start', 'end')
+    def valid_loc(cls, v):
+        assert v[0] == 'p' or v[0] == 'q', 'start/end must start with p or q'
+        return v
+
+class GenomicRegion(BaseModel):
+    type = 'computed'
```

Comment:
Is this correct?

Reply:
Yes, the first two attributes in GenomicRegion would be computed.

Reply:
Okay cool! Just wanted to double check.

Reply:
Echoing what I said on Linker -- should this have a `component_type` field?